File: online-judge-solutions/Codeforces/1694A.py
#https://codeforces.com/problemset/problem/1694/A
#Lang		:	Python 3.8
#Time		:	249 ms
#Memory         :	0 KB
for _ in range(int(input())):
    a,b=map(int,input().split())
    while(a>0 or b>0):
        if(a>0):
            print(0,end='')
            a-=1
        if(b>0):
            print(1,end='')
            b-=1
    print("\n",end="")

File: online-judge-solutions/Codeforces/1694A.cpp
//https://codeforces.com/problemset/problem/1694/A
//Lang		:	GNU C++14
//Time		:	30 ms
//Memory	:	0 KB
#include<iostream>
using namespace std;
int main(){
	int t,a,b;
	cin>>t;
	while(t--){
		cin>>a>>b;
		while(a>0 || b>0){
			if(a>0){cout<<0;a--;}
			if(b>0){cout<<1;b--;}
		}cout<<endl;
	}
	return 0;
}
